http_check/datadog_checks/http_check/http_check.py

```diff
@@ -208,6 +208,11 @@ def _load_conf(self, instance):
         instance_ca_certs = instance.get('ca_certs', self.ca_certs)
         weakcipher = _is_affirmative(instance.get('weakciphers', False))
         ignore_ssl_warning = _is_affirmative(instance.get('ignore_ssl_warning', False))
+
+        parsed_uri = urlparse(url)
+        if 'disable_ssl_validation' not in instance and parsed_uri.scheme == 'https' and not ignore_ssl_warning:
+            self.warning('Parameter disable_ssl_validation for {0} is not explicitly set, defaults to true'.format(url))
```

Comment:
Can we move this inside https://github.com/DataDog/integrations-core/blob/e591866622bdf24bc642974f1473c6000eb9ba87/http_check/datadog_checks/http_check/http_check.py#L245?

```python
if disable_ssl_validation and parsed_uri.scheme == "https" and not ignore_ssl_warning:
    if 'disable_ssl_validation' not in instance:
        self.warning('Parameter disable_ssl_validation for {0} is not explicitly set, defaults to true'.format(url))
    else:
        self.warning("Skipping SSL certificate validation for %s based on configuration" % addr)
```

Reply:
We could also put the second warning in an `else` to clean up logs, if you want.

Reply:
We could move the warning into _check(...), but I thought it might be better to keep the config logic in the _load_conf(...). I'm not opposed to the change, however.

As for your second point, yeah, it's a byproduct of where the config check is placed.

I'll make the changes 👍